### DISCUSSION TITLE
Couldn't make it work

### DISCUSSION THREAD

**Discussion by tojothethief** [zesterer/openmw-shaders#32] [Q&A]
Downloaded the devbuild from OMW's official site and followed the installation instructions for shaders, but the only difference I'm seeing in-game is extreme brightness. Not seeing the shaders in the shader menu either.

**Comment by zesterer** [comment]
Core shaders don't appear in the settings menu, only post-processing shaders appear there. If you edit some of the options in `config.glsl` and restart the game, do you see a change?

**Comment by tojothethief** [comment]
Thanks for the offered help. I uninstalled OMW for now but I hope to use it for my next character. I will see if that works whenever I get to it
